HW8/hw_8_task_4.py

```diff
@@ -0,0 +1,10 @@
+# 4. (необов'язкове виконання) Вивести всі елементу масиву, які є числом, використовуючи filter.
+
+lst = [1, 'cat', 2, 'dog', 3, 'rabbit', 4, 'cow', 5]
+
+def numbers(element):
+    if type(element) == int:
+        return element
```

Comment:
filter function returns a boolean value, so in this case, 0 value will be omitted. So it can be implemented like:
```suggestion
    return type(element) == int  # or nice function for checking type is isinstance(element, int)
```